contracts/Question.sol

```diff
@@ -78,9 +87,9 @@ contract Question {
 
   function answer(string _tweetUrl) beforeDeadline {
     require(msg.sender == backend);
-    tweetUrl = _tweetUrl;
     assert(token.transfer(charityAddress, donations));
-  }
+    tweetUrl = _tweetUrl;
+}
```

Comment:
Is order important here? As I understand, any exception in method execution will lead to reverting of the whole changes.